aether/sdk/drf/tests/test_views.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import random
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import RequestFactory
+from aether.sdk.unittest import UrlsTestCase
+from django.urls import reverse
+from rest_framework import status
+from aether.sdk.tests.fakeapp import models
+
+TEST_REALM = 'realm-test'
+
+
+class ViewsTest(UrlsTestCase):
+
+    def setUp(self):
+        super(ViewsTest, self).setUp()
+
+        username = 'user'
+        email = 'user@example.com'
+        password = 'secretsecret'
+        user = get_user_model().objects.create_user(username, email, password)
+
+        self.request = RequestFactory().get('/')
+        self.request.COOKIES[settings.REALM_COOKIE] = TEST_REALM
+        self.request.user = user
+
+        self.assertTrue(self.client.login(username=username, password=password))
+        self.client.cookies[settings.REALM_COOKIE] = TEST_REALM
+        test_names = ['one', 'two', 'three']
+        for i in range(3):
+            parent = models.TestModel.objects.create(name=test_names[i])
+            for _ in range(10):
+                models.TestChildModel.objects.create(
+                    name=str(random.randint(1, 100)),
+                    parent=parent,
+                )
+
+    def tearDown(self):
+        super(ViewsTest, self).tearDown()
+        models.TestModel.objects.all().delete()
```

Comment:
This is done automatically by the tearDown method always. It's not necessary to indicate it ;-)